test/run-tests

```diff
@@ -601,21 +573,21 @@ def check_commit_needs_testing(status, commands):
         return True
 
     # or the status is pending without a hostname in the description
-    if status["state"] == "pending" and not status.get("description"):
+    if status.state == "pending" and not status.description:
```

Comment:
the gh api will return None if you use `status.description` and the status has no `description` field?

Reply:
It returns an empty string, not None. Is it ok?
```
>>> for status in statuses:
...   if status.description == '':
...     print(status.state, 'empty')
...   else:
...     print(status.state, status.description)
<<snip>>
pending empty
failure linux-system-roles-staging-123-9blzw@2020-07-16 04:59:59.853483
```

Reply:
yes - so
```python
     if status.state == "pending" and not status.description:
```
is ok